src/erlcloud_json.erl

```diff
@@ -0,0 +1,45 @@
+-module(erlcloud_json).
```

Comment:
if it will be rejected to be merged (which I doubt), we can just move functionality inside of dirrect_connect itself. 

However, it's very useful change for erlcloud, since it can be used from every module that uses json. Currently kinesis & cloudtrail just returns `jsx:decode` results, and erlcloud_ddb2 does it's conversion to records, similar to what is done in `erlcoud_json.erl`